data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,57 @@ mission "Arachi Cartoonist 2"
 				accept
 			label noship
 			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`
+
+mission "Coalition Folklore: Saryd"
+	minor
+	invisible
+	to offer
+		random < 3
+	source
+		attributes "saryd" "folklore"
+	on offer
+		conversation
+			`Walking by the spaceport, you're approached by a Saryd in unusually traditional clothing, even by Saryd standards.`
+			`	"Greetings, Captain. Interested in participating in a cultural celebration, would you be?" He asks.`
+			choice
+				`	"What sort of celebration?"`
+				`	"Sorry, I'm busy right now."`
+					decline
+			`	"A celebration of Saryd folklore, it is," he explains. "Though for a long time among the stars we've lived, timeless classics from before our spacefaring age, we still celebrate. A set of theatrical plays about some of them, today we are hosting, and come watch them for free, anyone can."`
+			choice
+				`	"I've always got time for a story, lead the way."`
+				`	"I'm not much of a fan of plays, I'll pass."`
+					decline
+			`	You follow him to outside the spaceport city, and deep into a nearby forest. On the way there, several other Saryds in ancient garbs are guiding others who were also intrigued by the offer.`
+			`	You arrive to a clearing with three different paths to take, and you can make out a faint light deep into each of them. After a few minutes it seems all Saryds involved in the celebration have arrived, as they grab three caskets from behind a bush, and lay them before you and the others they brought here.`
+			`	"Now friends, choose your path, you must," one of the Saryds says, and many from the crowd grab small objects from the caskets, seemingly already familiar with this. You move forward to look into the caskets, and see that each one has different things inside: the left one is full of seeds, the middle one full of seashells, and the right one full of feathers.`
+			choice
+				`	(Grab a seed.)`
+					goto seed
+				`	(Grab a seashell.)`
+					goto pearl
+				`	(Grab a feather.)`
+			`	Once you have the feather in your hands, the organizers tell you to go through the path on the right, and so you do.`
+			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it, while keeping your eyes on the small stage in front of you. After the audience is all here, the curtains open, to reveal many small figures made out of what looks like paper, in a style that loosely resembles origami.`
```

Comment:
So I got through two of these and then it became clear that questions need to be asked.
Is this like a puppet show? How big exactly is this stage? A bonfire in the middle of a clearing is going to rather limit the field of space where the audience can get a view from, so is this a shadow play? 
There's a lot more that can be done here to visually set up the scene for the reader.